packages/pigeon/platform_tests/shared_test_plugin_code/lib/integration_tests.dart

```diff
@@ -979,6 +988,26 @@ void runPigeonIntegrationTests(TargetGenerator targetGenerator) {
       expect(api.callFlutterNoop(), completes);
     });
 
+    // TODO(tarrinneal): add flutter api error handling and enable these tests,
+    // See: https://github.com/flutter/flutter/issues/118243
```

Comment:
We generally put the TODOs on the skip line (even though it's ugly to autoformat) so it's very obvious where the explanation for the skip is.